src/cow-react/modules/swap/state/EthFlow/ethFlowInFlightOrderIds.ts

```diff
@@ -0,0 +1,19 @@
+import { atom } from 'jotai'
+import { atomWithStorage } from 'jotai/utils'
+
+export const ethFlowInFlightOrderIdsAtom = atomWithStorage<string[]>('eth-flow-in-flight-order-ids:v1', [])
```

Comment:
Nitpicks:

1. Start with v0?
2. Don't start with a version at all?
3. Use camelCase for storage key instead

Reply:
> Start with v0?

Fine with me. Done. Although, I would think normally APIs for example start in v1. I would start in v0 if we add the minor and patch. Like v0.0.1 or v0.1.0. But is not important. 

> Don't start with a version at all?

We added it for others. It's something Sasha suggested as a good practice, and I like it. First version I could skip adding the version, but, then 
* If we do changes, we might forget to version (easier to notice, if we that v1 in the name)
* We follow always the same pattern